core/client/src/test/java/alluxio/client/file/options/OutStreamOptionsTest.java

```diff
@@ -21,8 +21,10 @@
 import alluxio.client.file.policy.LocalFirstPolicy;
```

Comment:
is there no unit test for CreateUfsFileOptions / CompleteUfsFileOptions? if not, could you please add one and make sure your changes are included in the test? thanks